### PR TITLE
Add country and confirmed case to the UI

### PR DESCRIPTION
**Pull request by allysonjp715**
Requires a naming fix to the data-service. The tests need updating because there was a validator that the start date was after Oct 2019 (or a date close to that).

Also removes the table data refresh since "resolve()" does that for you.

### CODE REVIEW

**Comment by allysonjp715** [comment]
Something's going wrong and I'm unable to run the tests locally, and I'm failing to guess at the fixes :) I will investigate and fix tomorrow. Feel free to not review until then!

File: verification/curator-service/ui/src/components/LinelistTable.tsx [hunk 1]
```diff
@@ -131,11 +131,11 @@ export default class LinelistTable extends React.Component<{}, LinelistTableStat
             if (!this.validateRequired(newRowData.source_url)) {
                 return reject();
             }
+            console.log('here');
```
Comment (khmoran) [inline verification/curator-service/ui/src/components/LinelistTable.tsx:134]: nit: remove (& below)

this is my MO, i get it

File: verification/curator-service/ui/src/components/LinelistTable.tsx [hunk 2]
```diff
@@ -114,8 +116,6 @@ export default class LinelistTable extends React.Component<{}, LinelistTableStat
             let deleteUrl = this.state.url + rowData.id;
             const response = axios.delete(deleteUrl);
             response.then(() => {
-                // Refresh the table data
-                this.state.tableRef.current.onQueryChange();
                 resolve();
```
Comment (khmoran) [inline verification/curator-service/ui/src/components/LinelistTable.tsx:119]: can this be shortened to response.then(resolve)? (same above)

File: data-serving/data-service/src/model/case.ts [hunk 3]
```diff
@@ -135,7 +135,7 @@ const caseSchema = new mongoose.Schema(
                         type: String,
                         required: 'Enter a name for the event',
                     },
-                    date: dateRangeSchema,
+                    dateRange: dateRangeSchema,
```
Comment (khmoran) [inline data-serving/data-service/src/model/case.ts:138]: thanks for fixing! slowly working my way through refactoring + adding tests

File: verification/curator-service/ui/src/components/LinelistTable.tsx (hunk 1 above)
Comment (allysonjp715) [reply verification/curator-service/ui/src/components/LinelistTable.tsx]: :P 
done

File: verification/curator-service/ui/src/components/LinelistTable.tsx (hunk 2 above)
Comment (allysonjp715) [reply verification/curator-service/ui/src/components/LinelistTable.tsx]: Nice! Done.

File: data-serving/data-service/src/model/case.ts (hunk 3 above)
Comment (allysonjp715) [reply data-serving/data-service/src/model/case.ts:138]: nw!

**Pull request merged**